src/tables/extensions/data-table.component.tsx

```diff
@@ -9,35 +9,69 @@ import {
   TableHeader,
   TableHead,
   DataTableSkeleton,
+  InlineLoading,
 } from "carbon-components-react";
+import { useLayoutType } from "@openmrs/esm-framework";
+import styles from "./data-table.scss";
 
 interface InfoDataTable {
   headerData: [];
   rowData: [];
   isLoading: boolean;
+  tableTitle?: string;
 }
 
 const commonHeaderData = [
   {
     header: "Name",
     key: "name",
+    id: 0,
   },
   {
     header: "Value",
     key: "value",
+    id: 1,
   },
 ];
 
-const InfoDataTable = ({ rowData, isLoading, headerData }) => {
+const CardHeader = ({ isValidating, title, isTablet }) => {
+  return (
+    <div className={isTablet ? styles.tabletHeader : styles.desktopHeader}>
+      <h4>{title}</h4>
+      <span>{isValidating ? <InlineLoading /> : null}</span>
+    </div>
+  );
+};
+
+const InfoDataTable = ({
+  rowData,
+  isLoading,
+  headerData,
+  tableTitle = "",
+  isValidating,
+}) => {
+  const layout = useLayoutType();
+  const desktopView = layout === "desktop";
+  const isTablet = useLayoutType() === "tablet";
```

Comment:
It's a little weird to call the same hook twice in a row here, especially since you're already setting it to local variable `layout`

Reply:
Yah, I forgot that, I will fix it.